SourceCode/support/convert_masterfiles_to_csv.py

```diff
@@ -397,18 +398,25 @@ def get_model_classification(titles_path, variables_df):
 def convert_masterfiles_to_csv(models, gamma_overwrite_pos="not possible", overwrite_existing=False):
     """
     The main function to convert masterfiles to csv files. 
-    Depending on how you run it, it can have two types of behaviour:
+    Depending on how you run it, it can have three types of behaviour:
         a) If you run it as a script, it will overwrite files. It will
         ask for confirmation before overwriting the gamma values
-        b) If you call it from another function, it will only generate
-        new file if the files do not yet exist.
+        - If you call it from another function
+            b) It will generate files if files don't exist yet
+            c) It will overwrite files if the masterfiles are newer than the csv files
```

Comment:
Functionality b) and c) are working. However, a) is not and is overwriting even despite check

Reply:
issue seems to me to be in gamma_input_on_overwrite(out_dir, var, gamma_options) function. It's hard to tell what this is doing and I don't quite get the logic of the "gamma_options" argument?